test/accumulators_fraction_test.cpp

```diff
@@ -19,15 +19,25 @@ void run_tests() {
 
   // ctor
   {
-    f_t f(1, 2);
-    BOOST_TEST_EQ(f.successes(), 1);
-    BOOST_TEST_EQ(f.failures(), 2);
+    f_t f1(3, 1);
+    f_t f2(3.0, 5.0);
+    f_t f3(1.0, 3.0);
+    BOOST_TEST_EQ(f1.successes(), 3); BOOST_TEST_EQ(f1.failures(), 1); BOOST_TEST_EQ(f1.successes(), 3.0); BOOST_TEST_EQ(f1.failures(), 1.0);
+    BOOST_TEST_EQ(f2.successes(), 3); BOOST_TEST_EQ(f2.failures(), 5); BOOST_TEST_EQ(f2.successes(), 3.0); BOOST_TEST_EQ(f2.failures(), 5.0);
+    BOOST_TEST_EQ(f3.successes(), 1); BOOST_TEST_EQ(f3.failures(), 3); BOOST_TEST_EQ(f3.successes(), 1.0); BOOST_TEST_EQ(f3.failures(), 3.0);
```

Comment:
You do not need to compare to 3 **and** 3.0,  etc., this is guaranteed by C++. Please remove the superfluous tests.

Reply:
Otherwise this looks good.